content/NavCoin Core/How to move from NavPi to Raspberry Pi 4 and CM4.en.md

```diff
@@ -0,0 +1,182 @@
+---
+title: How to move from NavPi to Raspberry Pi 4 and CM4 for staking
+date: "2021-05-31"
+order: "000"
+---
+
+
+# Moving from NavPi to headless Raspberry Pi Compute Module 4 for staking
+
+Managing the NavPi through its GUI made managing and staking Navcoin a very simple process. Wanting to use the latest navcore 6.X and moving to a more up-to-date and powerful Raspberry Pi platform made staking more commandline heavy due to the missing GUI component when wanting to stake on a Pi remotely. 
+
+Below is a quick and dirty how to on how to move from a NavPi to a Raspberry Pi Compute Module 4 (CM4) and the latest version of navcoin 6.x to stake Navcoins headless on the Pi.
+
+**Disclaimer**: This how to was made to work with a lot of googeling and asking a ton of stupind questions to the patient Discord members of the Navcoin #navpi-support channel. It may not be the 'only' or 'best' way BUT this is how I was able to get staking working for me.
+
+**Special Thanks to `@salmonskinroll` and `@aguycalled` as well as `u/Zane_TLI` on Reddit for this [post](https://www.reddit.com/r/NavCoin/comments/l8jdcr/staking_with_navcoind_getstakinginfo_return>)**
+
+This how to is using the CM4 [CM4104032](https://www.buyapi.ca/product/raspberry-pi-compute-module-4-wireless-4gb-32gb-cm4104032/).
+
+1. Assumption is you have a running CM4 (how to can be found [here](https://www.youtube.com/watch?reload=9&v=jp_mF1RknU4)) and ssh access to it
+
+
+   **Notes**:
+
+- I had to use Raspbian 32bit as I believe the Navcoin binaries only come in 32-bit for arm (if I learn otherwise I will update)
+- I also am utilizing the `screen` window session manager to keep things running in case you get disconnected. [Here is a quick intro on how to use `screen`](https://www.youtube.com/watch?v=hB6Y72DK8mc)
+
+```
+pi@raspberrypi:~ $ screen
+```
+
+**Note**: When reconnecting to the Raspberry Pi remotely re-connect to the still running window session(s)
+
+```
+pi@raspberrypi:~ $ screen -x
+```
+
+          - I am performing all commands from the /home/pi directory
+          - At time of writing the latest version of navcoin was 6.1.0
+
+2. Get the latest Navcoin files from [here](https://build.nav.community/binaries/master/navcoin-6.1.0-arm-linux-gnueabihf.tar.gz)
+
+```
+pi@raspberrypi:~ $ wget https://build.nav.community/binaries/master/navcoin-6.1.0-arm-linux-gnueabihf.tar.gz
+```
+
+3. Once downloaded untar the tarball file 
```

Comment:
the numbering is not registered in the preview, not sure how to fix this but it's something related to formatting.